plugins/generator-1.16.4/forge-1.16.4/procedures/item_smelting_result.java.ftl

```diff
@@ -1,5 +1,5 @@
 <#include "mcitems.ftl">
 /*@ItemStack*/
-(world.getWorld().getRecipeManager().getRecipe(IRecipeType.SMELTING, new Inventory(${mappedMCItemToItemStackCode(input$item, 1)}), world.getWorld()).isPresent() ?
-        world.getWorld().getRecipeManager().getRecipe(IRecipeType.SMELTING, new Inventory(${mappedMCItemToItemStackCode(input$item, 1)}), world.getWorld()).get().getRecipeOutput().copy()
+(((World) world).getWorld().getRecipeManager().getRecipe(IRecipeType.SMELTING, new Inventory(${mappedMCItemToItemStackCode(input$item, 1)}), ((World) world).getWorld()).isPresent() ?

```

Comment:
.getWorld should not be used. Don't do it that way but do it the same way you did with other PRs please.

Applies to all procedures

Reply:
Sometimes, I have problems like here: `((World) world).getWorldInfo().getServer()`

Reply:
I don't think WorldInfo can provide server. Also I don't think you need worldinfo here.

Reply:
Procedure block removed from PR.